executor/src/witgen/jit/block_machine_processor.rs

```diff
@@ -570,7 +570,9 @@ params[3] = main_binary::C[3];"
     #[test]
     fn poseidon() {
         let input = read_to_string("../test_data/pil/poseidon_gl.pil").unwrap();
-        generate_for_block_machine(&input, "main_poseidon", 12, 4).unwrap();
+        generate_for_block_machine(&input, "main_poseidon", 12, 4)
+            .map_err(|e| eprintln!("{e}"))
```

Comment:
leftover `eprintln` or intended?

Reply:
I thought about removing it, but why not keep it?